integrations/datadog/.port/resources/port-app-config.yaml

```diff
@@ -102,3 +102,27 @@ resources:
             timeframe: .__timeframe
           relations:
             slo: .slo.id
+  - kind: serviceMetric
+    selector:
+      query: "true"
+      datadogSelector:
+        metric: "avg:system.mem.used"
+        env: '*'
+        service: '*'
+        timeframe: 10
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogServiceMetric"'
+          identifier: .__query_id
+          title: .query
+          properties:
+            query: .__query
+            series: .series
+            res_type: .res_type
+            from_date: '.from_date / 1000 | todate'
+            to_date: '.to_date / 1000 | todate'
+            metric_name: .__metric
+            env: .__env
+          relations:
+            service: .__service
```

Comment:
looks good, but I think this and the blueprint shouldn't be by default, as user wouldn't want to export system.mem.used by default, so lets create docs containing this example.

Reply:
also lets say user want to actually see the current value of the metric how would that be achieved

Reply:
It is a timeseries so the user can see the latest series by specifying 1 minute

Reply:
Let's update default to 1 min

Reply:
And have an example on how to actually map the value

Reply:
I have added the docs https://github.com/port-labs/port-docs/pull/1502